mesonbuild/interpreter/type_checking.py

```diff
@@ -25,7 +25,10 @@
 if T.TYPE_CHECKING:
     from typing_extensions import Literal
 
+    from .. import mparser
     from ..interpreterbase import TYPE_var
+    from ..interpreterbase.baseobjects import SubProject
+
```

Comment:
These imports got added in the wrong commit, BTW.

Reply:
This is not resolved. Now the imports are still in the wrong commit, but they get reverted in the next commit. :o